web/src/components/profile/ProfileAddressLockedFields.tsx

```diff
@@ -0,0 +1,31 @@
+import { AddressContext } from "@/contexts/addressContext";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ReactElement, useContext } from "react";
+
+export const ProfileAddressLockedFields = (): ReactElement => {
+  const { state } = useContext(AddressContext);
+
+  const Config = getMergedConfig();
+  return (
+    <div className="margin-top-4 margin-bottom-4" data-testid={"locked-profileAddressFields"}>
+      <div className="text-bold margin-bottom-05">
+        {Config.profileDefaults.fields.businessAddress.default.header}
+      </div>
+
+      {state.addressData.addressLine1 &&
```

Comment:
pebble - why are we validating the fields have a value? is it needed? If something was missing, all a user would see is the business address header. I belive you're controlling whether this element is rendered at a higher level.

Reply:
Your controlling whether this renders by `areAddressFieldsDisabled()` in the ProfileNewJerseyAddress component

Reply:
Without the conditional rendering, when a user does not enter an address, once the user forms, the address renders as `undefined, NJ`. Please see JP's feedback: https://github.com/newjersey/navigator.business.nj.gov/pull/8293#discussion_r1704431345

I verified with Product and they want the business address section to render empty. This logic allows it to render as such.

Reply:
@AnJuHyppolite Should this check be higher up, say on line 10? Is there any reason we need to show the section header if the section itself will be empty?